apps/web/src/components/execution-detail/ExecutionDetailsStepHeader.tsx

```diff
@@ -99,7 +103,6 @@ const generateDetailByStepAndStatus = (status, job) => {
   if (job.type === StepTypeEnum.DELAY) {
     const { digest, step: stepMetadata, payload } = job;
 
-    if (!digest.amount && !digest.unit) return `Waiting to receive execution delay from bridge endpoint`;
```

Comment:
i was not able to find why we checked digest on the delay step and how this bug/regression was created.